### ISSUE TITLE
In-Memory Databases

### ISSUE THREAD

**Issue by bricelam** [aspnet/Microsoft.Data.Sqlite.Docs#8]
It's probably worth re-iterating how in-memory database connections work. Interesting points:
* How to access the same in-memory database from multiple connections (named, shared cache)
* The lifetime of in-memory databases

**Comment by Rick-Anderson** [comment]
@bricelam  If you can get this doc (and the others) drafted, I can help finish it off.  cc/ @tdykstra

**Comment by tdykstra** [comment]
Should this issue be in the [EF docs](https://github.com/aspnet/entityframework.docs/) repo?

**Comment by bricelam** [comment]
No, but all of the System.Data.Sqlite documentation could probably be linked to from there.

**Comment by bricelam** [comment]
See also: [Shareable In-Memory SQLite Databases](http://www.bricelam.net/2015/12/10/shareable-in-memory-databases.html)
